client/src/components/register-login/register.js

```diff
@@ -52,7 +52,7 @@ class Register extends Component {
       showStrong: false,
       passwordStrength: '',
       passwordInputStarted: false,
-      passwordInputTitle: "Password shouldn't be weak and less than 8 symbols",
+      passwordInputTitle: 'We recommend setting strong password with 8 or more symbols',
```

Comment:
You should explain user when he or she wrongs. This message still doesn't have any information about start or ending with spaces. You can use separate message in that specific case

Reply:
I have 10
![image](https://user-images.githubusercontent.com/6152066/80943168-40146500-8def-11ea-9a21-8259e475788f.png)

Reply:
Done